### PR TITLE
(1126b) Use Refer paths in Refer case lists

### PR DESCRIPTION
**Pull request by yndajas**
## Context

Previously we were using the Assess paths everywhere for the link to view an individual referral

## Changes in this PR

- Allow Refer paths to be used for links to referrals in the Refer case lists

## Release checklist

[Release process documentation](../doc/how-to/perform-a-release.md)

As part of our continuous deployment strategy we must ensure that this work is
ready to be released once merged.

### Pre-merge

- [ ] There are changes required to the Accredited Programmes API for this change to work...
  - [ ] ... and they have been released to production already

### Post-merge

- [ ] This adds, extends or meaningfully modifies a feature...
  - [ ] ... and I have written or updated an end-to-end test for the happy path in the [Accredited Programmes E2E repo](https://github.com/ministryofjustice/hmpps-accredited-programmes-e2e)
- [ ] This makes new expectations of the API...
  - [ ] ... and I have notified the API developer(s) of changes to the contract tests (Pact), or the API is already compliant
- [ ] [Manually approve](../doc/how-to/perform-a-release.md#releasing-to-the-preprod-environment) release to preprod
- [ ] [Manually approve](../doc/how-to/perform-a-release.md#releasing-to-the-production-environment) release to prod

### CODE REVIEW

File: server/utils/referrals/caseListUtils.ts
Comment (Gweaton) [inline server/utils/referrals/caseListUtils.ts:109]: I'd be tempted to remove the default parameter here, so there's no risk of people forgetting it and/or interpreting a priority of one over the other

Comment (yndajas) [reply server/utils/referrals/caseListUtils.ts:109]: We'd need to provide it even when it's irrelevant (i.e. for any other column) - do you think that's better than accidentally using a default?

**Comment by yndajas** [comment]
[Remove "Date referred" column from Assess case list (S)](https://trello.com/c/GuYDs740/1261-remove-date-referred-column-from-assess-case-list-s)

File: server/utils/referrals/caseListUtils.ts
Comment (Gweaton) [reply server/utils/referrals/caseListUtils.ts:109]: Hmm, I forgot how widespread our use of `tableRowContent` was! I wonder if we should have a look at breaking it down slightly as a refactoring task, rather than blocking this PR.

**Review by Gweaton — APPROVED** [review]

**Pull request merged**
